docs/_source/docs/stack_config.rst

```diff
@@ -100,6 +101,48 @@ hooks
 A list of arbitrary shell or Python commands or scripts to run. Find out more
 in the :doc:`hooks` section.
 
+
+.. _launch_action:
+
+launch_action
+~~~~~~~~~~~~~
+* Resolvable: No
+* Can be inherited from StackGroup: Yes
+* Inheritance strategy: Overrides parent if set
+
+This setting determines whether or not the stack should be included or excluded when running
+``sceptre launch``. This must be one of two values: "include" or "exclude". The default value for
+this (which doesn't need to be specified) is "include". This **only** applies to the ``launch``
+command.
+
+If the ``launch_action`` is set to ``"exclude"``, it means that:
+ * If the stack does NOT exist, the Stack will not be created and will be skipped over
+ * If the stack *does* currently exist, it will be deleted.
```

Comment:
i'm not a fan of coupling these together in one param.  what if user just wants to skip but not actually delete?

Reply:
What would you think about adding an enum for "skip" that neither creates nor deletes?